README.md

```diff
@@ -17,9 +17,16 @@ The **[Databricks Lakehouse](https://www.databricks.com/)** provides one simple
 
 # dbt-databricks
 
-The `dbt-databricks` adapter contains all of the code enabling dbt to work with Databricks.
+The `dbt-databricks` adapter contains all of the code enabling dbt to work with Databricks. This adapter is based off the amazing work done in [dbt-spark](https://github.com/dbt-labs/dbt-spark). Some key features include:
 
-This adapter is based off the amazing work done in [dbt-spark](https://github.com/dbt-labs/dbt-spark)
+- **Easy setup**. No need to install an ODBC driver as the adapter uses pure Python APIs.
+- **Open by default**. For example, it uses the the open and performant [Delta](https://delta.io/) table format by default. This has many benefits, including letting you use `MERGE` as the the default incremental materialization strategy.
+- **Performance**. The adapter generates SQL expressions that are automatically accelerated by the native, vectorized [Photon](https://databricks.com/product/photon) execution engine.
+
```

Comment:
Should we also add 3-level namespace support with Unity Catalog as a key feature?

Reply:
Good idea! Done.